QUARKUS-5615.md

```diff
@@ -0,0 +1,30 @@
+## Links to the resources
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5615
+- Upstream documentation PR: https://github.com/quarkusio/quarkus/pull/46182
+- Follow-up to the original PR: https://github.com/quarkusio/quarkus/pull/46211
+- Serialisation PR: https://github.com/quarkusio/quarkus/pull/41063
+- Serialisation documentation PR: https://github.com/quarkusio/quarkus/pull/42289
```

Comment:
Add one more level to have serializers related content in one sub-tree and deserializers in another one

Reply:
Changed the structure